lib/c/detect_cgroup_path.c

```diff
@@ -85,6 +85,15 @@ char* detect_cgroup_path(const char* controller, int user_id, const char* id) {
         return path;
     }
 
+    // Try cgroups v2 with systemd but non-slice mountpoints and with cgroup name instead of container id (used for debug purposes)
+    snprintf(path, PATH_MAX,
+             "/sys/fs/cgroup/system.slice/%s/%s",
```

Comment:
I thought more that the string should be "%s/%s" to make it fully flexible. This is now restricted to groups in system.slice ?

Reply:
you're right 😊 I changed it now